python/pshexs/influxdb.py

```diff
@@ -0,0 +1,122 @@
+
+# import influxdb
+from pshconfig import Config
+import traceback, sys
+
+
+def test_output():
+
+    # Create a new Config object to ease reading the Platform.sh environment variables.
+    # You can alternatively use os.environ yourself.
+    config = Config()
+
+    # The 'database' relationship is generally the name of primary SQL database of an application.
+    # That's not required, but much of our default automation code assumes it.' \
+    # credentials = config.credentials('database')
+
+    try:
+
+        credentials = config.credentials('influxdb')
+
+        return credentials
+
+
+    except Exception as e:
+        return traceback.format_exc(), sys.exc_info()[0]
+
+
+
+# def test_output():
+#     return '{"influxdb_test": "passed"}'
```

Comment:
I presume all of this is going to be deleted?

Reply:
Yessir - just updated that one.